### DISCUSSION TITLE
US Census Tract numbers

### DISCUSSION THREAD

**Discussion by ryan-morris** [osm-search/Nominatim#3478] [General]
Is it possible to retrieve the US Census Tract ID/Number from Nominatim? I've imported the TIGER dataset but I'm not sure if that is contained there. I have some required reporting that includes that information and all I have available is latitude and longitude. I've tried including &extratags=1 in the requests to reverse geocode but nothing extra comes back

For reference, this is what I'm referring to: https://www2.census.gov/geo/pdfs/education/CensusTracts.pdf

**Comment by mtmail** [comment]
I'm afraid they don't contain the tract numbers you're looking for. Nominatim only imports road segments for address search.

The https://github.com/osm-search/TIGER-data/ scripts convert TIGER EDGE files into house number range, street number, state, zip code and line-string geometry. You can see a small example here https://github.com/osm-search/TIGER-data/blob/master/tests/fixtures/expected_37143.csv Nominatim imports those converted *.csv files.

**Reply by ryan-morris** [reply]
Thanks for that information, I'll look at tracking that outside of OSM/nominatim
